pkg/diagnostics/cluster/roles.go

```diff
@@ -20,7 +20,34 @@ type ClusterRoles struct {
 }
 
 const (
-	ClusterRolesName = "ClusterRoles"
+	ClusterRolesName   = "ClusterRoles"
+	clusterRoleMissing = `
+clusterrole/%s is missing.
+
+Use the 'oadm policy reconcile-cluster-roles' command to create the role. For example,
+
+  $ oadm policy reconcile-cluster-roles \
```

Comment:
I could argue that bit of the CLI hacking guide is only about the command usage statement. I think diagnostics is the only place that has output related to running other commands entirely. It's a gray area; I think it is clearer with the `$` to indicate commands; what do you think?